content-src/asrouter/templates/SimpleBelowSearchSnippet/_SimpleBelowSearchSnippet.scss

```diff
@@ -91,7 +91,11 @@
     }
 
     @media (max-width: 865px) {
-      margin: 0 60px 0 0;
+      inset-inline-end: 60px;
```

Comment:
This probably could just be removed if there's nothing special to handle at this breakpoint?

Reply:
You're right that that's not the right rule, I checked again; but the rule isn't actually the same as the above, since what we want at that breakpoint is to push the snippet in line with the search bar, by having the margin only at the end of the line (so 0 60  0 0 instead of 0 60, which would put 60px at both the start and end). 

EDIT: I think what we need is `margin-inline-end`, plus a margin of 0 to keep the next breakpoint from kicking in.